fabric-convention-tags-v2/src/client/java/net/fabricmc/fabric/impl/tag/convention/ConventionLogWarningsClient.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.tag.convention;
+
+import java.util.List;
+
+import it.unimi.dsi.fastutil.objects.ObjectArrayList;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import net.minecraft.client.resource.language.I18n;
+import net.minecraft.item.Item;
+import net.minecraft.registry.Registry;
+import net.minecraft.registry.RegistryKeys;
+import net.minecraft.registry.tag.TagKey;
+import net.minecraft.util.Identifier;
+
+import net.fabricmc.api.ClientModInitializer;
+import net.fabricmc.fabric.api.event.lifecycle.v1.ServerLifecycleEvents;
+import net.fabricmc.loader.api.FabricLoader;
+
+public class ConventionLogWarningsClient implements ClientModInitializer {
+	public static final Logger LOGGER = LoggerFactory.getLogger(ConventionLogWarningsClient.class);
+
+	/**
+	 * A config option mainly for developers.
+	 * Logs out modded item tags that do not have translations when running on integrated server.
+	 * Defaults to SILENCED.
+	 */
+	private static final String LOG_UNTRANSLATED_WARNING_MODE = System.getProperty("fabric-tag-conventions-v2.missingTagTranslationWarning", LogWarningMode.SILENCED.name());
+	private enum LogWarningMode {
+		SILENCED,
+		DEV_SHORT,
+		DEV_VERBOSE
```

Comment:
Drop the dev prefix, I dont see why its needed as you could do SHORT or VERBOSE in prod?

Reply:
We have a check for dev environment only. It's what I copied from ConventionLogWarning.java class as I was told it was desired that legacy tag warning to only be possible in dev.
![image](https://github.com/FabricMC/fabric/assets/40846040/e078934e-47a2-45c0-9811-5300e6290c24)

But if we are fine with these being usable in prod, I can drop the environment check and rename the enums. (Should I also change ConventionLogWarning.java as well or leave that?)

Reply:
nvm. Will make changes. I'm silly